src/main/java/org/apache/commons/imaging/formats/png/chunks/PngChunkText.java

```diff
@@ -42,9 +47,9 @@ public PngChunkText(final int length, final int chunkType, final int crc, final
         final int textLength = bytes.length - (index + 1);
         text = new String(bytes, index + 1, textLength, StandardCharsets.ISO_8859_1);
 
-        if (getDebug()) {
-            System.out.println("Keyword: " + keyword);
-            System.out.println("Text: " + text);
+        if (LOGGER.isLoggable(Level.FINE)) {
```

Comment:
Uhg, the JUL level names are AWFUL :=(

Reply:
I went straight for `LOGGER.isDebugEnabled()` and the IDE remembered me what it actually looked like :=/